draft-ietf-mls-protocol.md

```diff
@@ -2229,14 +2229,32 @@ Proposals. It instructs group members to update their representation of the
 state of the group by applying the proposals and advancing the key schedule.
 
 Each proposal covered by the Commit is identified by a ProposalID value, which
```

Comment:
Need to update a few references of ProposalID to ProposalOrRefType;

Reply:
Good point, thanks.